### PR TITLE
PP Prep: Removing PG, updated SDK, refactored name of extension

### PR DESCRIPTION
**Pull request by temandr**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/dms_sync/azext_dms/_help.py [hunk 1]
```diff
@@ -32,16 +31,16 @@
 
 helps['dms project task create'] = """
     type: command
-    short-summary: Create a migration Project which can contain multiple Tasks.
+    short-summary: Create and start a migration Task.
```
Comment (navinnarra) [inline src/dms_sync/azext_dms/_help.py]: nit: Why do we need to capitalize 'Task'?

File: src/dms_sync/azext_dms/_help.py [hunk 2]
```diff
@@ -32,16 +31,16 @@
...
     parameters:
         - name: --source-platform
           type: string
           short-summary: >
-            The type of server for the source database. The supported types are: SQL, PostgreSQL, MySQL
+            The type of server for the source database. The supported types are: SQL, MySQL
```
Comment (navinnarra) [inline src/dms_sync/azext_dms/_help.py]: Nit: Put a '.' at the end to be consistent with other comments.

File: src/dms_sync/azext_dms/vendored_sdks/datamigration/models/connect_to_source_oracle_task_properties.py [hunk 3]
```diff
@@ -52,14 +49,13 @@ class ConnectToSourceOracleTaskProperties(ProjectTaskProperties):
...
-        'client_data': {'key': 'clientData', 'type': '{str}'},
         'task_type': {'key': 'taskType', 'type': 'str'},
         'input': {'key': 'input', 'type': 'ConnectToSourceOracleTaskInput'},
         'output': {'key': 'output', 'type': '[ConnectToSourceNonSqlTaskOutput]'},
     }
 
-    def __init__(self, client_data=None, input=None):
-        super(ConnectToSourceOracleTaskProperties, self).__init__(client_data=client_data)
+    def __init__(self, input=None):
+        super(ConnectToSourceOracleTaskProperties, self).__init__()
```
Comment (navinnarra) [inline src/dms_sync/azext_dms/vendored_sdks/datamigration/models/connect_to_source_oracle_task_properties.py]: Why are we including this?

Comment (temandr) [reply src/dms_sync/azext_dms/vendored_sdks/datamigration/models/connect_to_source_oracle_task_properties.py]: Didn't upstream my fork. Good catch

File: src/dms_sync/azext_dms/_help.py (hunk 1 above)
Comment (temandr) [reply src/dms_sync/azext_dms/_help.py]: It's already a convention we follow in the GA version of the DMS CLI tool.

Comment (navinnarra) [reply src/dms_sync/azext_dms/_help.py]: Got it.

File: src/dms_sync/azext_dms/_help.py [hunk 4]
```diff
@@ -32,16 +31,16 @@
...
         - name: --source-platform
           type: string
           short-summary: >
-            The type of server for the source database. The supported types are: SQL, PostgreSQL, MySQL
+            The type of server for the source database. The supported types are: SQL, MySQL.
         - name: --target-platform
           type: string
           short-summary: >
-            The type of service for the target database. The supported types are: SQLDB, AzureDbForPostgreSql, AzureDbForMySql.
+            The type of service for the target database. The supported types are: SQLDB, AzureDbForMySql.
```
Comment (navinnarra) [inline src/dms_sync/azext_dms/_help.py]: Server?

**Pull request merged**